bedrock/mozorg/templates/mozorg/about/governance/policies/module-ownership.html

```diff
@@ -0,0 +1,66 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "mozorg/about-base.html" %}
+
+{% block page_title %}{{ _('Mozilla Modules and Module Owners') }}{% endblock %}
+{% block body_id %}module-ownership{% endblock %}
+
+{% block article %}
+<h1 class="title-shadow-box">{{ self.page_title() }}</h1>
+<p>{{ _('The Mozilla project is far too big for any one person &mdash; or even a small set of people &mdash; to make the decisions. Instead, decision-making is distributed to a range of participants through the project being split into “<a href="%s">Modules</a>”, each of which has a Module Owner. A module is, in the case of code, a piece of functionality, or in the case of non-code, an activity, with reasonably well-defined boundaries. (Some of the below explanation is code-focussed, but analagous points can be made for non-code.)')|format('https://wiki.mozilla.org/Modules') }}</p>
```

Comment:
Nit: we can probably forgo using the HTML entity `&mdash;` here since we're using utf-8?